src/Acf.php

```diff
@@ -185,4 +199,23 @@ private static function apply_default_transform( $field_obj ) {
 	private static function get_transform_class_name( $field_type ) {
 		return '\\' . __NAMESPACE__ . '\\Acf\\Transforms\\' . str_replace( '_', '', ucwords( $field_type, '_' ) );
 	}
+
+	/**
+	 * Get an ACF group
+	 *
+	 * @param $group_id
+	 * @return bool
+	 */
+	private static function get_group( $group_id ) {
+		// @codingStandardsIgnoreStart
+		// Ignore use WP_Query rule because we don't know if this will be a sub-query and hence create complications.
```

Comment:
On the performance side I'm not really sure what's perform better but I think if we rely on WP to do the query I think is a little bit better at least on the maintenance side. And about the CI this is what we did on Printmore, https://github.com/moxienyc/Printmornyc.com/blob/master/wp-content/themes/Printmornyc/codesniffer.ruleset.xml#L16 since those CI applies are required by the VIP env.